packages/sdk/src/types/attributes/requests/ExecuteIdentityAttributeQueryRequest.ts

```diff
@@ -1,6 +1,3 @@
-export interface ExecuteIdentityAttributeQueryRequest {
-    valueType: string;
-    validFrom?: string;
-    validTo?: string;
-    tags?: string[];
-}
+import { IdentityAttributeQuery } from "../IdentityAttributeQuery";
+
+export interface ExecuteIdentityAttributeQueryRequest extends IdentityAttributeQuery {}
```

Comment:
Hm, müsste es in dem Request nicht eine `query` Property geben? Der entsprechende Typ in der Runtime sieht jedenfalls so aus:

```ts
export interface ExecuteIdentityAttributeQueryRequest {
    query: IIdentityAttributeQuery | IdentityAttributeQueryJSON;
}
```

Reply:
nein, der request wird in die query gegeben
```ts
public async executeRelationshipAttributeQuery(query: any): Promise<Envelope> {
    const result = await this.consumptionServices.attributes.executeRelationshipAttributeQuery({ query });
    return this.ok(result);
}
```

Reply:
Nach persönlicher Absprache: wird in einem anderen PR angepasst